about.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="src/css/mobile-styles.css" />
+    <link rel="stylesheet" href="src/css/about-mobile-styles.css" />
+    <link rel="stylesheet" href="src/css/desktop-styles.css" />
+    <link rel="stylesheet" href="src/css/about-desktop-styles.css" />
+  </head>
+  <body>
+    <nav class="socials">
+      <img
+        src="https://img.icons8.com/ios-filled/20/000000/twitter.png"
+        alt="twitter"
+      />
+      <img
+        src="https://img.icons8.com/ios-filled/20/000000/facebook.png"
+        alt="facebook"
+      />
+      <p>English</p>
+      <p>My Page</p>
+      <p>Log out</p>
+    </nav>
+    <header class="app-header">
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+
+      <img
+        class="main-logo"
+        src="src/images/logos/riscv-color.svg"
+        alt="website"
+        onclick="window.location.href = 'index.html'"
+      />
+
+      <nav class="mobile-navbar">
+        <ul class="mobile-nav-menu">
+          <li><a href="about.html" class="nav-item">About</a></li>
+          <li><a href="#" class="nav-item">Program</a></li>
+          <li><a href="#" class="nav-item">Join</a></li>
+          <li><a href="#" class="nav-item">Sponsor</a></li>
+          <li><a href="#" class="nav-item">News</a></li>
+          <li><a href="#" class="nav-item">CC Campaign</a></li>
+        </ul>
+
+        <ul class="desktop-nav-menu">
+          <li><a href="about.html" class="nav-item">About</a></li>
+          <li><a href="#" class="nav-item">Program</a></li>
+          <li><a href="#" class="nav-item">Join</a></li>
+          <li><a href="#" class="nav-item">Sponsor</a></li>
+          <li><a href="#" class="nav-item">News</a></li>
```

Comment:
- [ ] It would be better to add the Home page link to the navbar of the about page so the user can navigate from the about page to the home page.